app/views/users/articles_written.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>user.name 's' Articles</h1>
```

Comment:
If you are going to display information from Ruby objects such as `user`, they need to be surrounded with these: `<%= %>` like so: `<%= user.name %>`

Reply:
I did not mean to push this file oops.